File: .tmp_py4013/batch_11_dump/3238_find_the_number_of_winning_players.js
// LeetCode 3238 - Find the Number of Winning Players
// https://leetcode.com/problems/find-the-number-of-winning-players/

var winningPlayerCount = function(n, pick) {
    const cnt = Array.from({length: n}, () => new Array(11).fill(0));
    const s = new Set();
    for (const p of pick) {
        const x = p[0], y = p[1];
        cnt[x][y]++;
        if (cnt[x][y] > x) s.add(x);
    }
    return s.size;
};

File: .tmp_rs4013/rust_check15/3238_find_the_number_of_winning_players.rs
struct Solution;
// LeetCode 3238 - Find the Number of Winning Players
// https://leetcode.com/problems/find-the-number-of-winning-players/

use std::collections::HashSet;

impl Solution {
    pub fn winning_player_count(n: i32, pick: Vec<Vec<i32>>) -> i32 {
        let mut cnt = vec![[0; 11]; n as usize];
        let mut s = HashSet::new();
        for p in pick {
            let x = p[0] as usize;
            let y = p[1] as usize;
            cnt[x][y] += 1;
            if cnt[x][y] > x as i32 {
                s.insert(x);
            }
        }
        s.len() as i32
    }
}

fn main() {}
